group_vars/all/ah_group_roles.yml

```diff
@@ -0,0 +1,14 @@
+---
+# Documentation: https://github.com/ansible/galaxy_collection/blob/devel/roles/group_roles/README.md
+ah_group_roles:
```

Comment:
I think you need to provide it with [] value or uncomment the example

Reply:
@djdanielsson right! I added it, thanks for the review!